From f2fb923e6920f567033e0954195e5c59e38aa164 Mon Sep 17 00:00:00 2001
From: Jan Przybylak <jplx256@gmail.com>
Date: Sun, 16 Dec 2018 14:29:34 -0500
Subject: [PATCH 14/18] Removed background gradient

Summary:
Removed a background gradient that's impossible to render properly on most screens.
Basically the same as D16998

Reviewers: #vdg, dantti, ndavis

Reviewed By: #vdg, ndavis

Subscribers: ndavis, dantti

Differential Revision: https://phabricator.kde.org/D17341
---
 libapper/CategoryDrawer.cpp | 29 -----------------------------
 1 file changed, 29 deletions(-)

diff --git a/libapper/CategoryDrawer.cpp b/libapper/CategoryDrawer.cpp
index 5e0a939..d12f265 100644
--- a/libapper/CategoryDrawer.cpp
+++ b/libapper/CategoryDrawer.cpp
@@ -49,35 +49,6 @@ void CategoryDrawer::drawCategory(const QModelIndex &index,
     const QFontMetrics fontMetrics = QFontMetrics(font);
     const int height = categoryHeight(index, option);
 
-    //BEGIN: decoration gradient
-    {
-        QPainterPath path(optRect.bottomLeft());
-        path.lineTo(QPoint(optRect.topLeft().x(), optRect.topLeft().y() - 3));
-        const QPointF topLeft(optRect.topLeft());
-        QRectF arc(topLeft, QSizeF(4, 4));
-        path.arcTo(arc, 180, -90);
-        path.lineTo(optRect.topRight());
-        path.lineTo(optRect.bottomRight());
-        path.lineTo(optRect.bottomLeft());
-
-        QColor window(option.palette.window().color());
-        const QColor base(option.palette.base().color());
-
-        window.setAlphaF(0.4);
-
-        QLinearGradient decoGradient1(optRect.topLeft(), optRect.bottomLeft());
-        decoGradient1.setColorAt(0, window);
-        decoGradient1.setColorAt(1, Qt::transparent);
-
-        QLinearGradient decoGradient2(optRect.topLeft(), optRect.topRight());
-        decoGradient2.setColorAt(0, Qt::transparent);
-        decoGradient2.setColorAt(1, base);
-
-        painter->fillPath(path, decoGradient1);
-        painter->fillPath(path, decoGradient2);
-    }
-    //END: decoration gradient
-
     {
         QRect newOptRect(optRect);
         newOptRect.setLeft(newOptRect.left() + 1);
-- 
2.20.1

